sda/cmd/s3inbox/healthchecks.go

```diff
@@ -66,7 +66,9 @@ func (h *HealthCheck) RunHealthChecks() {
 }
 
 func (h *HealthCheck) httpsGetCheck(url string, timeout time.Duration) healthcheck.Check {
-	cfg := &tls.Config{MinVersion: tls.VersionTLS12}
+	cfg := &tls.Config{
+		MinVersion: tls.VersionTLS12,
+	}
```

Comment:
```suggestion
	cfg := &tls.Config{MinVersion: tls.VersionTLS12}
```

We should be consistent across all files, either a one liner or verbose writing.

Reply:
Fixed.